addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -1,3 +1,95 @@
+// ======= VARIABLES =======
+$o-wblog-read-width: 620px;
+$o-wblog-read-container-sidebar-width :900px;
+$o-wblog-loader-size: 50px;
```

Comment:
Ideally, variables should be declared in a variables scss file (see website for example) with the `!default` flag. They can be set here to BS value expression if needed (still with the `!default` flag).

Reply:
I may use [website_blog/static/src/scss/primary_variables.scss ](https://github.com/odoo/odoo/blob/ac64a5b697ce47b94d21d279608d970563841ed3/addons/website_blog/static/src/scss/primary_variables.scss), it looks quite empty at the moment.

Is it ok?